src/Hocon.Configuration/HoconConfigurationFactory.cs

```diff
@@ -59,10 +59,29 @@ public static Config ParseString(string hocon)
         ///     The configuration defined in the configuration file. If the section
         ///     "akka" is not found, this returns an empty Config.
         /// </returns>
-        [Obsolete("Call the ConfigurationFactory.Default method instead.")]
```

Comment:
Didn't realize this had been done - needs to be reverted. Can't just change a behavioral contract on a minor version like this.